apps/performance/default-onpush/src/app/person-list.component.ts

```diff
@@ -1,39 +1,18 @@
 import { Component, Input } from '@angular/core';
-
-import { CDFlashingDirective } from '@angular-challenges/shared/directives';
 import { CommonModule } from '@angular/common';
 import { FormsModule } from '@angular/forms';
 import { MatChipsModule } from '@angular/material/chips';
-import { MatFormFieldModule } from '@angular/material/form-field';
-import { MatInputModule } from '@angular/material/input';
 import { MatListModule } from '@angular/material/list';
 
 @Component({
   selector: 'app-person-list',
   standalone: true,
-  imports: [
-    CommonModule,
-    FormsModule,
-    MatListModule,
-    MatFormFieldModule,
-    MatInputModule,
-    MatChipsModule,
-    CDFlashingDirective,
-  ],
+  imports: [CommonModule, FormsModule, MatListModule, MatChipsModule],
```

Comment:
warning; you deleted the CDFlashingDirective. So you think the application is well optimized because CD is not flashing, but that's not the case.